### ISSUE TITLE
Don't singularize "news"

### ISSUE THREAD

**Issue by sascha-seyfert** [cundd/rest#6]
"Could not analyse class:News\Domain\Repository\NewRepository"

rest/Classes/Cundd/Rest/DataProvider/Utility.php->singularize.

Maybe you can add an TS-option to prevent this behavior.

**Comment by cundd** [comment]
Or should I add a dictionary of words that will not be singularized, because 'news' should never be converted - I guess

**Comment by sascha-seyfert** [comment]
:thumbsup:

**Comment by cundd** [comment]
I added a static list of words to skip when converting to singular. Not the most flexible solution so far, but it should be a start.

Would be nice if you could test it (beside the Unit Test).

**Comment by cundd** [comment]
This should be fixed with the TypoScript configuration in e71db9d2fbf9a6c4700ea245b1bec9e667cd996a.

```typoscript
plugin.tx_rest.settings {
	# Define words that should not be converted to singular
	singularToPlural {
		news = news
		equipment = equipment
		information = information
		rice = rice
		money = money
		species = species
		series = series
		fish = fish
		sheep = sheep
		press = press
		sms = sms
	}
}
```

Feel free to reopen if you think this is still an issue.
